custom_components/tuya_local/devices/forceclima_soundless.yaml

```diff
@@ -48,16 +60,27 @@ primary_entity:
           value: medium
         - dps_val: high
           value: high
+
     - id: 20
-      type: integer
       name: fault_code
+      type: bitfield
+      mapping:
+        - dps_val: 1
+          value: E1
+        - dps_val: 2
+          value: E2
+        - dps_val: 3
+          value: warter_full
```

Comment:
As per previous comment, this should be attached to the problem sensor below, but I would also suggest adding a mapping to the sensor of `dps_val: 4` to `value: false`, and add an additional binary_sensor with `translation_key: tank_full` with a mapping of `dps_val: 4` to `value: true` and default mapping of `value: false`

Reply:
This one I don't understand what I have to change

Reply:
OK, I will take care of this before merge